_data/other.yml

```diff
@@ -246,7 +246,7 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
-      doc: https://support.formstack.com/customer/portal/articles/1833071-two-factor-authentication
+      doc: https://support.formstack.com/customer/portal/articles/1833071
```

Comment:
Link doesn't work for me. Just me?

Reply:
It seems they changed it completely to
https://help.formstack.com/hc/en-us/articles/360019517891

Reply:
```suggestion
      doc: https://help.formstack.com/hc/en-us/articles/360019517891
```